---
layout: default
title:  'Gymrek Lab - Software'
---

<h4>Check out our lab <a href="https://github.com/gymreklab" target="_blank">github page</a> for more info on our software projects.</h4>

<table class="softwaretable">
<tbody>
<tr>
	<td width=50%>
		<h2>lobSTR</h2>
		lobSTR is a tool for profiling Short Tandem Repeats (STRs) from high throughput sequencing data. It takes as input raw sequencing reads in either BAM or FASTQ format and outputs STR genotypes in a VCF file. lobSTR can be used to simultaneously genotype thousands of samples.
		<br>
		[<a href="http://www.ncbi.nlm.nih.gov/pubmed/22522390" target="_blank">paper</a>]
		[<a href="lobstr.teamerlich.org" target="_blank">website</a>]
		[<a href="https://github.com/mgymrek/lobstr-code" target="_blank">github</a>]
	</td>
	<td width=50%>
		<h2>PyBamView</h2>
		PyBamView is a python-based application that allows you to view BAM sequence alignments through your web browser. It was inspired by the samtools tview tool, and was motivated by the need to better visualize alignments with complex insertions/deletions across multiple samples, which is not handled well by current broswers such as IGV and the UCSC Genome Browser.
		<br>
		[<a href="http://www.ncbi.nlm.nih.gov/pubmed/25147359" target="_blank">paper</a>]
		[<a href="http://melissagymrek.com/pybamview/" target="_blank">website</a>]
		[<a href="https://github.com/mgymrek/pybamview" target="_blank">github</a>]
	</td>
</tr>
</tbody>
</table>