---
layout: default
title:  'Gymrek Lab - Personal Genomics'
---

<h1>Personal Genomics for Bioinformaticians - Winter 2017</h1>

[<a href="https://s3-us-west-2.amazonaws.com/cse291personalgenomics/genome_course_syllabus_winter17.pdf">course syllabus</a>]

<h2>Course description</h2>
Genome-sequencing is quickly becoming a commodity, and more than a million people have already analyzed their own genomes through direct-to-consumer companies. This course provides an introduction to current bioinformatics techniques for analyzing and interpreting human genomes. We will learn how to interpret a single genome in the context of an entire population, based on the often quoted concept: interpreting one genome requires tens of thousands of genomes. Topics covered include an introduction to human medical and population genetics, human ancestry, finding and interpreting disease-causing variants, genome-wide association studies, genetic risk prediction, analyzing next generation sequencing data, and how to scale current genomics techniques to analyze hundreds of thousands of genomes. We will also discuss the social impact of the personal genomics revolution.

<h2>Schedule</h2>
<table class="schedule">
	<tr>
		<th>Module</th>
		<th>Date</th>
		<th>Lecture</th>
		<th>Homework</th>
	</tr>
	<tr>
	<td>Introduction to personal genomics</td>
	<td>01-10 (Th)</td>
	<td><a href="#l1">Introduction to your genome</a></td>
	<td><a href="ps1_resources.html">ps1 out</a></td>
	</tr>
	<td>What can I do with my genome?</td>
	<td>01-12 (Th)</td>
	<td><a href="#l3">Basic population genetics</a></td>
	<td><a href="ps2_resources.html">ps2 out</a></td>
	</tr>
	<tr>
	<td></td>
	<td>01-17 (T)</td>
	<td>Determining ancestry</td>
	<td></td>
	</tr>
	<tr>
	<td></td>
	<td>01-19 (Th)</td>
	<td>Phasing and imputation</td>
	<td></td>
	</tr>
	<tr>
	<td></td>
	<td>01-24 (T)</td>
	<td>Web 2.0 Genomics</td>
	<td></td>
	</tr>
	<tr>
	<td></td>
	<td>01-26 (Th)</td>
	<td>Social impact of personal genomics</td>
	<td><a href="ps2_resources.html">ps2 due</a></td>
	</tr>

	<tr>
	<td>Complex traits</td>
	<td>01-31 (T)</td>
	<td>Introduction to GWAS</td>
	<td><a href="ps3_resources.html">ps3 out</a></td>
	</tr>
	<tr>
	<td></td>
	<td>02-02 (Th)</td>
	<td>Guest lecture TBD</td>
	<td></td>
	</tr>
	<tr>
	<td></td>
	<td>02-07 (T)</td>
	<td>Advanced GWAS topics</td>
	<td><a href="project_resources.html">project out</a></td>
	</tr>
	<tr>
	<td></td>
	<td>02-09 (Th)</td>
	<td>Scaling GWAS to millions of genomes</td>
	<td><a href="ps3_resources.html">ps3 due</a></td>
	</tr>

	<tr>
	<td>Next-gen sequencing and big data</td>
	<td>02-14 (T)</td>
	<td>Introduction to NGS</td>
	<td><a href="ps4_resources.html">ps4 out</a></td>
	</tr>
	<tr>
	<td></td>
	<td>02-16 (Th)</td>
	<td>Short read alignment and variant calling</td>
	<td><a href="project_resources.html">proposal due</a></td>
	</tr>
	<tr>
	<td></td>
	<td>02-21 (T)</td>
	<td>Functional genomics - RNAseq, ChIPseq</td>
	<td></td>
	</tr>
	<tr>
	<td></td>
	<td>02-23 (Th)</td>
	<td>Storing, querying, visualization</td>
	<td></td>
	</tr>
	<tr>
	<td></td>
	<td>02-28 (T)</td>
	<td>Long read sequencing technologies</td>
	<td><a href="ps4_resources.html">ps4 due</a></td>
	</tr>

	<tr>
	<td>Mutation hunting</td>
	<td>03-02 (Th)</td>
	<td>Introduction to genetic mapping</td>
	<td><a href="ps5_resources.html">ps5 out</a></td>
	</tr>
	<tr>
	<td></td>
	<td>03-07 (T)</td>
	<td>Filtering and prioritizing variants</td>
	<td><a href="ps5_resources.html">ps5 due</a></td>
	</tr>
	<tr>
	<td></td>
	<td>03-09 (Th)</td>
	<td>De novo mutations and constraint</td>
	<td></td>
	</tr>
	<tr>
	<td></td>
	<td>03-14 (T)</td>
	<td>Interpreting non-coding variants</td>
	<td></td>
	</tr>

	<tr>
	<td>Final project</td>
	<td>03-16 (Th)</td>
	<td>Project presentations</td>
	<td><a href="project_resources.html">project due</a></td>
	</tr>

</table>

<h2>Resources for problem sets</h2>
<ul>
<li><a href="using_comet.html">Getting started with comet</a></li>
<li><a href="using_23andme.html">Notes on using 23andme data for problem sets</a></li>
<li><a href="ps1_resources.html">Problem set 1</a></li>
<li><a href="ps2_resources.html">Problem set 2</a></li>
<li><a href="ps3_resources.html">Problem set 3</a></li>
<li><a href="ps4_resources.html">Problem set 4</a></li>
<li><a href="ps5_resources.html">Problem set 5</a></li>
<li><a href="project_resources.html">Final project</a></li>
</ul>

<h2>Class material</h2>

<a name="l1"></a>
<h3>Lecture 1: Introduction to your genome</h3>
<ul>
	<li>Slides</li>
	<li>Lecture Notes</li>
</ul>

<a name="l2"></a>
<h3>Lecture 2: Social impact of personal genomics</h3>
<ul>
	<li>Slides</li>
	<li>Lecture Notes</li>
	<li>Recommended reading</li>
	<ul>
		<li><a href="http://science.sciencemag.org/content/339/6117/321">Identifying personal genomes by surname inference</a></li>
	</ul>
</ul>

<a name="l3"></a>
<h3>Lecture 3: Basic population genetics</h3>
<ul>
	<li>Slides</li>
	<li>Lecture Notes</li>
	<li>Recommended reading</li>
	<ul>
		<li><a href="http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html">A global reference for human genetic variation</a></li>
	</ul>
</ul>


